content/en/docs/how-tos/interact-with-running-jobs.md

```diff
@@ -0,0 +1,146 @@
+to access the cluster against which the extended test suites are running.
+Access the `Terminal` tab of the running test pod (`Workloads` -> `Pods` -> `Terminal`). Find the `kubeconfig` file in the pod's `/tmp` directory.
+Once copied to your local system, you can proceed to run `oc` commands against the cluster by setting `KUBECONFIG` environment variable
+or passing `--kubeconfig` to `oc`. Again, this is intended for WIP PRs only. The cluster will be terminated whenever the job completes.
+It is usually more productive to launch a cluster using `cluster-bot` through `Slack` and manually run `openshift-tests` suites against that, rather
+than through a PR job's cluster. For how to run `openshift-tests` binary and to find more information about the test suites, see
+[How to run the test suites myself outside of CI.](#how-to-run-the-test-suites-myself-outside-of-ci)
+
+## How do I know which tests will run?
+
+It can be quite confusing to find the test command that is running in CI. Jobs are configured in
+[the release repository](https://github.com/openshift/release). To give an example, the `e2e-*` tests run the command
+[openshift-tests run "${TEST_SUITE}" ${TEST_ARGS:-}](https://github.com/openshift/release/blob/8ccb2b1c17387920b3de0180c52052c535603855/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh#L142)
+against external clusters. You can find other test commands in
+[release/ci-operator/config/openshift](https://github.com/openshift/release/tree/master/ci-operator/config).
+For example, the `ci/prow/unit` test command for `openshift-apiserver` is
+[make test-unit JUNITFILE=${ARTIFACT_DIR}/junit_report.xml --warn-undefined-variables](https://github.com/openshift/release/blob/8ccb2b1c17387920b3de0180c52052c535603855/ci-operator/config/openshift/openshift-apiserver/openshift-openshift-apiserver-master.yaml#L60-#L62). The job definitions are
+generated from the step-registry workflow and/or the ci-operator/config files. For example, with the `openshift-apiserver` repository, the job yaml
+definitions are found
+[here](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/openshift-apiserver/openshift-openshift-apiserver-master-presubmits.yaml)
+You might grep for `e2e-aws` or `ci/prow/unit` to view an individual job yaml.
+
+## How to run the test suites myself outside of CI
+
+While iterating upon a WIP PR, for those jobs that install and run tests against an external cluster, it's useful to run the `openshift-tests` binary
+against a test cluster you have running rather than follow a PR job in the CI cluster.
+You can run the `openshift-tests` binary against any cluster for which you have a kubeconfig file.
+The test binary can be built from [openshift/origin repository](https://github.com/openshift/origin) and all of the `e2e` individual tests are found
+in [origin/test/extended](https://github.com/openshift/origin/tree/master/test/extended). First, clone the origin repository with:
```

Comment:
If they're just grabbing the latest from `master` they could also pull `registry.ci.openshift.org/ocp/4.7:tests` and use that, right? Or is that less useful? (IDK lol I don't do this)

Reply:
yup, they could - but i figured if they are modifying the tests/adding a case, the'll want to use the code - I can put a suggestion for the tests image, too

Reply:
done, added the tests image example